superset/tags/commands/create.py

```diff
@@ -67,25 +67,23 @@ def validate(self) -> None:
 
 class CreateCustomTagWithRelationshipsCommand(CreateMixin, BaseCommand):
     def __init__(self, data: dict[str, Any], bulk_create: bool = False):
-        self._tag = data["name"]
-        self._objects_to_tag = data.get("objects_to_tag")
-        self._description = data.get("description")
+        self._properties = data.copy()
         self._bulk_create = bulk_create
 
     def run(self) -> None:
         self.validate()
 
         try:
-            tag = TagDAO.get_by_name(self._tag.strip(), TagTypes.custom)
-            if self._objects_to_tag:
-                TagDAO.create_tag_relationship(
-                    objects_to_tag=self._objects_to_tag,
-                    tag=tag,
-                    bulk_create=self._bulk_create,
-                )
+            tag_name = self._properties["name"]
```

Comment:
why is name accessed directly as opposed to the other properties that are using .get?

Reply:
required vs. not required, right now with the current PR name is the only required to create the object.

Let me know if you'd want me to just use .get for everything

Reply:
no, this makes sense, thanks!